app/assets/stylesheets/my-app/_use_mixins.scss

```diff
@@ -0,0 +1,3 @@
+section {
+  @include linear-gradient(to top, red, orange);
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color `orange` should be written in hexadecimal form as `#ffa500`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `orange` should only be used in variable declarations; they should be referred to via variable everywhere else.